### PR TITLE
Week 2 Progress: Tan Min Rong Samson

### PR DESCRIPTION
**Pull request by samsontmr**

### CODE REVIEW

File: students/AY1617S2/tanMinRongSamson/TanMinRongSamson-Progress.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
+# Tan Min Rong Samson - Weekly Progress
+
+---
+
+### Week 2
+
+Effort| Type | PR | Issue
+:----:|:----:|:-----------|:------
+1 | D | [`#6334`](https://github.com/TEAMMATES/teammates/pull/6334) | Update feedback entry count to 5 million
+3 | D | [`#6338`](https://github.com/TEAMMATES/teammates/pull/6338) | Rename html files in InstructorFeedbackResultsPageUiTest
```
Comment (whipermr5) [inline students/AY1617S2/tanMinRongSamson/TanMinRongSamson-Progress.md]: I think the effort values can only be powers of 2

Comment (samsontmr) [reply students/AY1617S2/tanMinRongSamson/TanMinRongSamson-Progress.md]: oops, thanks!

**Comment by damithc** [comment]
@samsontmr there is a merge conflict that needs to be resolved.

**Comment by samsontmr** [comment]
@damithc resolved, thanks!

**Pull request merged**